Fixing typo!

wking-io opened the pull request:
Feel free to do whatever you want, but wanted to provide a suggestion to make it as easy as possible for you to click and deploy 🤘

kentcdodds commented:
Thanks David! Someone pinged me about this on twitter before I noticed your PR and I've already fixed it. Sorry about that!

kentcdodds commented:
@all-contributors please add @davidluhr for content

wking-io committed:
Fixing typo!

Feel free to do whatever you want, but wanted to provide a suggestion to make it as easy as possible for you to click and deploy 🤘

kentcdodds reviewed (approved):
Thanks!
